### PR TITLE
ref #348: move shop relevant sidebar menu items and categories to sho…

### PR DESCRIPTION
**Pull request by pixeljunkie**
…p bundle

| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#348
| License       | MIT

Needs to be merged with the same branch in chameleon-base

### CODE REVIEW

File: src/ShopBundle/Bridge/Chameleon/Migration/Script/update-1549299700.inc.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<h1>Build #1549299700</h1>
```
Comment (UlrichKu) [inline src/ShopBundle/Bridge/Chameleon/Migration/Script/update-1549299700.inc.php]: I would probably give these updates totally different numbers.

(1549299689 +1 would be 1549299690 :-) )

Comment (pixeljunkie) [reply src/ShopBundle/Bridge/Chameleon/Migration/Script/update-1549299700.inc.php]: After a test. I changed the numbers to one before the core updates, so they will run first and require the core updates.

**Comment by UlrichKu** [comment]
There are build errors?

**Comment by pixeljunkie** [comment]
After a lot of testing in a non shop setup i found some problems with the main menu migration updates and fixed them. Now it is ready for merge.

**Pull request merged**